documentation/slate/source/includes/apis.md

```diff
@@ -1716,6 +1716,69 @@ object. Arrivals are ordered chronologically in ascending order.
 
 they are exactly the same as [departures](#departures)
 
+<a name="line-reports"></a>Line reports
+---------------------------------------
+
```

Comment:
It could be nice to have the same kind of json example that's available for traffic_reports.